linter.py

```diff
@@ -12,5 +16,57 @@ class Phpmd(Linter):
     tempfile_suffix = 'php'
     defaults = {
         'selector': 'source.php, text.html.basic',
-        '@rulesets:,': 'cleancode,codesize,controversial,design,naming,unusedcode'
```

Comment:
This must stay. All user settings should be declared like that. a) for (automated) documentation, b) to have correct quoting, and c) we support arrays as input automatically.

Reply:
I had to remove the `@` prefix, because it was adding appending the configured filename (`phpmd.xml` in my case) after the resolved file name. Is it okay to directly modify `self.defaults['rulesets']`?